include/crocoddyl/core/actuation-base.hpp

```diff
@@ -71,6 +68,17 @@ class ActuationModelAbstractTpl {
   virtual void calc(const boost::shared_ptr<ActuationDataAbstract>& data, const Eigen::Ref<const VectorXs>& x,
                     const Eigen::Ref<const VectorXs>& u) = 0;
 
+  /**
+   * @brief Compute the actuation signal from the state point only
+   *
+   * It updates the actuation signal based on the state only. This function is commonly used in
+   * the terminal nodes of an optimal control problem.
```

Comment:
I'm probably missing something here, but I don't get why we should need to compute the actuation signal in the terminal node, since in the terminal node there's no actuation defined in the OCP.

Reply:
We defined the actuation signal as a function of `x` and `u`.
Currently, we have implemented actuation models that depends on `u` only.

The x dependency is a sensible approach as it could help issues such as https://github.com/loco-3d/crocoddyl/issues/991 (see Pinocchio issues as well).

This is not conventional choice, but yet useful.

Is this answer your question? Can I mark this conversation as resolved?

Reply:
No, you didn't answer my question. :)
I get it that the actuation signal can depend on both x and u, that's a good idea. What I don't get is why we would need an actuation signal at all for the terminal nodes, since the dynamics doesn't have to be computed and you only need to evaluate the terminal cost.

Reply:
You're right.
I have fixed this, and soon I will push it.